src/mame/orca/vastar.cpp

```diff
@@ -1,6 +1,10 @@
 // license:BSD-3-Clause
 // copyright-holders: Allard van der Bas
 
+// Game plate and ROM provided by Ignacio Seki. Mame emulator code by Alberto Salso
+// here is the ROM
+// https://www.mediafire.com/file/r0qft5t6z1jpvk6/akazukin.zip/file
```

Comment:
No rom links in source file

Reply:
Agreed, best not to expose the ROM’s webpage of origin while others would search via Google or Bing.